tests/tracking/fluent/test_fluent.py

```diff
@@ -169,13 +147,13 @@ def create_experiment(
 
 
 @pytest.fixture(autouse=True)
-def reset_experiment_id():
+def reset_experiment_id(monkeypatch):
     """
     This fixture resets the active experiment id *after* the execution of the test case in which
     its included
     """
     yield
-    HelperEnv.set_values()
+    monkeypatch.delenvs((MLFLOW_EXPERIMENT_ID.name, MLFLOW_EXPERIMENT_NAME.name), raising=False)
```

Comment:
We can remove this line. `monkeypatch` restore the environment variables after each test finishes.

Reply:
you're right